### ISSUE TITLE
URAP: remove xmp files

### ISSUE THREAD

**Issue by lbestock** [arch-kiosk/arch-kiosk-office#2871]
There are an awful lot of tentaculaire xmp files floating around in URAP that should be removed.

**Comment by urapadmin** [comment]
U R A P`?

**Comment by lbestock** [comment]
Yeah. I was in there a bunch this morning getting stuff for my lecture on excavation and there were all these great octopi whom, however delightful, I don't really want in there:
![image](https://github.com/user-attachments/assets/2a9c009c-1913-49c7-a7b1-73fedef34e58)

**Comment by urapadmin** [comment]
you know how to bulk-delete files, no?

**Comment by urapadmin** [comment]
a filter by file extension would be a general extension that can come in handy at times...

**Comment by lbestock** [comment]
Yes the ticket was really just to remind myself to do it at some point, not a request. Though now you say it, a filter by extension WOULD come in handy...

**Comment by urapadmin** [comment]
- #2873
don't wait for it :o)

**Comment by luizaogs** [comment]
I did this using the fancy extension filter. 

(Yes, I know I'm crazy. And yes, I'm playing the fun game when I try to avoid writing my book by closing old tickets.)
